### ISSUE TITLE
Possible Bug in the Gradient Estimation Step L71

### ISSUE THREAD

**Issue by ghost** [jparkerholder/ASEBO#4]
According to Algorithm 1 from the NeurIPS paper, the [gradient estimation step](https://github.com/jparkerholder/ASEBO/blob/master/asebo/es.py#L71) should be ```g /= (2 * n_samples * params['sigma'])``` rather than ```g /= (2 * params['sigma'])```. In the current code, ```n_samples``` (i.e., ```n_t``` in pseudo-code) is missing. However, I am not very sure.

Thanks to @jparkerholder

**Comment by jparkerholder** [comment]
Hey - thank you for this. You are right. Technically we should do this. I think that in reality it still works because we normalize the gradients in train.py:

```
gradient /= (np.linalg.norm(gradient) / master.N + 1e-8)
```

which does a similar thing. This is an implementation detail I found to be helpful for all ES methods. Also there will be a trade-off with the learning rate. So I think that is why the code still works/is very close to the exact algorithm, but you are correct in that technically it is not getting there the most principled way.

Hope this helps!
